addons/live-featured-project/script.js

```diff
@@ -75,32 +77,20 @@ export default async function ({ addon, msg }) {
     const usp = new URLSearchParams();
     if (autoPlay) usp.set("autoplay", "");
     if (enableTWAddons) usp.set("addons", enabledAddons.join(","));
+    if (shareUsername) usp.set("username", username);
     iframeElement.setAttribute("src", `https://turbowarp.org/${projectId}/embed?${usp}`);
     wrapperElement.dataset.player = "turbowarp";
     if (!showMenu) iframeElement.setAttribute("height", "260");
   };
 
-  const loadForkphorus = () => {
-    wrapperElement.dataset.player = "forkphorus";
-    iframeElement.setAttribute(
-      "src",
-      `https://forkphorus.github.io/embed.html?id=${projectId}&auto-start=${autoPlay}&ui=${showMenu}`
-    );
-  };
-
   // Start loading the players
 
-  if (forceAlternative && alternativePlayer !== "none") {
-    if (alternativePlayer === "turbowarp") loadTurboWarp();
-    else if (alternativePlayer === "forkphorus") loadForkphorus();
-  } else {
-    loadScratch();
-    iframeElement.addEventListener("load", () => {
-      if (iframeElement.contentDocument.querySelector(".not-available-outer") !== null) {
-        if (alternativePlayer === "turbowarp") loadTurboWarp();
-        else if (alternativePlayer === "forkphorus") loadForkphorus();
-        else stageElement.removeChild(wrapperElement);
-      }
-    });
-  }
+  if (player === "turbowarp") loadTurboWarp();
+  else loadScratch();
+  iframeElement.addEventListener("load", () => {
+    if (iframeElement.contentDocument.querySelector(".not-available-outer") !== null) {
```

Comment:
Can you put this in a try, catch or find a way that doesn't throw an error when the project is shared?